addons/MagicaVoxelImporter/VoxelMesh.gd

```diff
@@ -29,14 +29,13 @@ func _set_neck_height(value):
 			mat.set_shader_param('neck_height', value)
 
 func set_render_head(value):
+	render_head = value
 	if is_inside_tree():
-		render_head = value
-		var vox_char_node := $"." as MeshInstance
-		var mat = vox_char_node.get_surface_material(0)
+		var mat = get_surface_material(0)
 		if !mat:
-			mat = vox_char_node.mesh.surface_get_material(0).duplicate()
+			mat = self.mesh.surface_get_material(0).duplicate()
```

Comment:
could omit the self keyword since it's assumed in gdscript (unlike python)
> The self variable is always available and is provided as an option for accessing class members, but is not always required 

https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html#functions